### PR TITLE
Update Year 2018 -> 2019 

### PR DESCRIPTION
**Pull request by MuhaddiMu**
Glad to be a contributor :smile:

### CODE REVIEW

**Comment by exoego** [comment]
Thanks for opening this PR.
Could you also update copyright year in other files ?
E.g. https://github.com/kazupon/vue-i18n/blob/ccf29f88d343e9baf2e6a5a9c04ad992c4f5906d/vuepress/zh/README.md

File: vuepress/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ home: true
 heroImage: ./vue-i18n-logo.png
 actionText: Get Started →
 actionLink: introduction.md
-footer: MIT Licensed | Copyright © 2018 kazuya kawaguchi
+footer: MIT Licensed | Copyright © 2019 kazuya kawaguchi
```
Comment (exoego) [inline vuepress/README.md]: Thanks for opening this PR.
Could you also update copyright year in other files ?
E.g. https://github.com/kazupon/vue-i18n/blob/ccf29f88d343e9baf2e6a5a9c04ad992c4f5906d/vuepress/zh/README.md

Comment (MuhaddiMu) [reply vuepress/README.md]: Yeah Sure!

**Comment by MuhaddiMu** [comment]
I updated in other files. 
Kindly let me know if I'm missing some files otherwise it's ready to merge

**Pull request merged**